controllers/authentication/openidconnect_controller.go

```diff
@@ -67,8 +77,33 @@ func (r *OpenIDConnectReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		algs = append(algs, string(alg))
 	}
 
+	var caBundle dynamiccertificates.CAContentProvider
+	if config.Spec.CABundle != nil {
+		caBundle, err = dynamiccertificates.NewStaticCAContent("CABundle", config.Spec.CABundle)
+		if err != nil {
+			log.Info("Invalid CABundle")
+
+			r.handlers.Delete(req.Name)
+
+			return reconcile.Result{}, err
```

Comment:
As I mentioned from above - you should not return any error here as it would cause the controller to put this OIDC into a rate-limited queue.